services/slam/client_test.go

```diff
@@ -28,23 +31,27 @@ import (
 	"go.viam.com/rdk/vision"
 )
 
-var (
-	nameSucc = "viam"
-	nameFail = "maiv"
+const (
+	nameSucc               = "viam"
+	nameFail               = "maiv"
+	chunkSizeInternalState = 2
+	chunkSizePointCloud    = 100
 )
 
 func TestClientWorkingService(t *testing.T) {
 	logger := golog.NewTestLogger(t)
 	listener, err := net.Listen("tcp", "localhost:0")
 	test.That(t, err, test.ShouldBeNil)
-
 	workingServer, err := rpc.NewServer(logger, rpc.WithUnauthenticated())
 	test.That(t, err, test.ShouldBeNil)
-
 	pose := spatial.NewPose(r3.Vector{X: 1, Y: 2, Z: 3}, &spatial.OrientationVector{Theta: math.Pi / 2, OX: 0, OY: 0, OZ: -1})
 	pSucc := referenceframe.NewPoseInFrame("frame", pose)
 	pcSucc := &vision.Object{}
 	pcSucc.PointCloud = pointcloud.New()
+	pcdPath := artifact.MustPath("slam/mock_lidar/0.pcd")
+	pcd, err := os.ReadFile(pcdPath)
+	test.That(t, err, test.ShouldBeNil)
+
 	err = pcSucc.PointCloud.Set(pointcloud.NewVector(5, 5, 5), nil)
 	test.That(t, err, test.ShouldBeNil)
 	imSucc := image.NewNRGBA(image.Rect(0, 0, 4, 4))
```

Comment:
using small byte slice for internal state stream tests because reasons

Reply:
Full pcds were used to confirm the creation of a pointcloud object from the returned pcd